app/components/Feedback/index.tsx

```diff
@@ -39,48 +40,69 @@ const Feedback = ({
   upHint,
   downHint,
   options,
+  formClassName,
   onSubmit,
 }: FeedbackProps) => {
-  const [showFeedback, setShowFeedback] = useState(false)
+  const [showThanks, setShowThanks] = useState(false)
   const [showFeedbackForm, setShowFeedbackForm] = useState(false)
+  const [vote, setVote]: any = useState(undefined)
```

Comment:
this means that each time the page gets refreshed, the user can vote for the article again. The previous solution would rememeber whether an article was liked or not between refreshes

Reply:
very good point. I've updated the code, but note that I had to pass a setVoted function to the child Action components so that the feedback knew to set the other Action disabled. Its the best I could do. Also the handler function for voting in the questions.actions file is throwing an error, and thus the feedback button is returning to its null state after you press it.